public/docs/ts/latest/guide/attribute-directives.jade

```diff
@@ -89,167 +92,165 @@ include ../_quickstart_repo
     There is also less risk of colliding with a third-party directive name when we give ours a prefix.
 
     We do **not** prefix our `highlight` directive name with **`ng`**.
-    That prefix belongs to Angular and
-    we don't want to confuse our directives with their directives.
+    That prefix belongs to Angular.
 
     We need a prefix of our own, preferably short, and `my` will do for now.
-    
+p
+  | After the `@Directive` metadata comes the directive's controller class, which contains the logic for the directive. 
+  +ifDocsFor('ts')
+    | We export `HighlightDirective` to make it accessible to other components.
 :marked
-  After the `@Directive` metadata comes the directive's controller class which we are exporting
-  to make it accessible to other components. 
-  The directive's controller class contains the logic for the directive.
-  
   Angular creates a new instance of the directive's controller class for
   each matching element, injecting an Angular `ElementRef` 
   into the constructor.
-  
   `ElementRef` is a service that grants us direct access to the DOM element
   through its `nativeElement` property.
   That's all we need to set the element's background color using the browser DOM API.
 
 .l-main-section
+a#apply-directive
 :marked
   ## Apply the attribute directive
   The `AppComponent` in this sample is a test harness for our `HighlightDirective`.
   Let's give it a new template that 
-  applies the directive as an attribute to a `span` element.
-  In Angular terms, the `<span>` element will be the attribute **host**.
-  
-  We'll put the template in its own `app.component.html` file that looks like this:
+  applies the directive as an attribute to a paragraph (`p`) element.
+  In Angular terms, the `<p>` element will be the attribute **host**.
+p
+  | We'll put the template in its own 
+  code #[+adjExPath('app.component.html')]
+  | file that looks like this:
 +makeExample('attribute-directives/ts/app/app.component.1.html',null,'app/app.component.html')(format=".")
 :marked
   A separate template file is clearly overkill for a 2-line template. 
   Hang in there; we're going to expand it later.
   Meanwhile, we'll revise the `AppComponent` to reference this template.
 +makeExample('attribute-directives/ts/app/app.component.ts',null,'app/app.component.ts')
 :marked
-  We've added an `import` statement to fetch the 'Highlight' directive and 
-  added that class to a `directives` array in the component metadata so that Angular 
+  We've added an `import` statement to fetch the 'Highlight' directive and, 
```

Comment:
The original problem is that the sentence was a [run on sentence](https://en.wikipedia.org/wiki/Run-on_sentence). Adding the comma, was a "quickfix" to the reader so that he/she knows to take their breath :). The sentence could be reworked, but just adding the comma seems good enough for now.